### ISSUE TITLE
Inaccuracy in 'crunching' even 64 bit numbers

### ISSUE THREAD

**Issue by chaitannah** [craigmayhew/bigprimes.net#22]
When queried with 2^63 - 1 (9223372036854775807) cruncher reports it to be almost certainly prime although it's prime factors are 7^2 · 73 · 127 · 337 · 92737 · 649657

Could you help me understand how your algorithm works so fast (with a good accuracy) ?

**Comment by craigmayhew** [comment]
Hello @chaitannah,

Sadly, when I added this function I failed to add any notes. I can see the algorithm looks flawed and this has remained unnoticed due to two things:
 - Inadequate unit tests here: https://github.com/craigmayhew/bigprimes.net/blob/master/test/Primes.php, introduced in this commit https://github.com/craigmayhew/bigprimes.net/commit/df1311137a06de94ef347761f9a46cfc456dbe16#diff-5e95248b71ef03bd8e8d5133d8609a74
 - A prime lookup is used for numbers up to 1400000000 rather than the erroneous algorithm. Numbers over this are not properly checked for primality due to the error you have found.

I will correct this in the upcoming overhaul of the site, unless someone submits a PR in the meantime.

Thanks for flagging the issue.
